dandi/tests/test_upload.py

```diff
@@ -100,16 +91,14 @@ def test_upload_locks(local_docker_compose_env, mocker, monkeypatch, tmp_path):
     lockmock.return_value.__exit__.assert_called()
 
 
-def test_upload_unregistered(local_docker_compose_env, monkeypatch, tmp_path):
-    DIRNAME = "sub-anm369963"
-    FILENAME = "sub-anm369963_ses-20170228.nwb"
+def test_upload_unregistered(local_docker_compose_env, monkeypatch, organized_nwb_dir):
+    nwb_file, = organized_nwb_dir.glob("*/*.nwb")
+    dirname = nwb_file.parent.name
     dandi_instance_id = local_docker_compose_env["instance_id"]
-    (tmp_path / dandiset_metadata_file).write_text("identifier: '999999'\n")
-    (tmp_path / DIRNAME).mkdir(exist_ok=True, parents=True)
-    copyfile(DANDIFILES_DIR / DIRNAME / FILENAME, tmp_path / DIRNAME / FILENAME)
-    monkeypatch.chdir(tmp_path)
+    (organized_nwb_dir / dandiset_metadata_file).write_text("identifier: '999999'\n")
+    monkeypatch.chdir(organized_nwb_dir)
     with pytest.raises(ValueError) as excinfo:
-        upload(paths=[DIRNAME], dandi_instance=dandi_instance_id, devel_debug=True)
+        upload(paths=[dirname], dandi_instance=dandi_instance_id, devel_debug=True)
     assert str(excinfo.value) == (
```

Comment:
for some reason coverage marks all code here as "red" as it is not run on CI -- we will need to double check.